debian/control

```diff
@@ -0,0 +1,31 @@
+Source: nntrainer
+Section: libs
+Priority: optional
+Maintainer: Jijoong Moon <jijoong.moon@samsung.com>
+Build-Depends: gcc-9 | gcc-8 | gcc-7 | gcc-6 | gcc-5 (>=5.4),
+ ninja-build, meson (>=0.50), debhelper (>=9),
+ libblas-dev, libiniparser-dev
+Standards-Version: 0.0.1
+Homepage: https://github.com/nnstreamer/nntrainer
+
+Package: nntrainer
+Architecture: any
+Multi-Arch: same
+Depends: ${shlibs:Depends}, ${misc:Depends}
+Description: Training Neural Network Models on Devices.
+ This is software framework for training neural network model on devices
+
+Package: nntrainer-dev
+Architecture: any
+Multi-Arch: same
+Depends: nntrainer
+Description: NNtrainer development package
+ This is development package for nntrainer.
+
+Package: nntrainer-applications
+Architecture: any
+Multi-Arch: same
+Depends: nntrainer, tesnorflow-dev, jsoncpp, libcurl, ${shlibs:Depends}, ${misc:Depends}
```

Comment:
Does it depend on tensorflow-dev? In addition, for jsoncpp and libcurl, you just need to add libjsoncpp-dev and libcurl[3|4]-[gnutls|nss|openssl]-dev in Build-Depends section above.

Reply:
Fixed.! Thanks!